app/models/ability.rb

```diff
@@ -4,6 +4,9 @@ class Ability # rubocop:disable Metrics/ClassLength
   include CanCan::Ability
 
   def initialize(compte)
+    droits_comptes_refuses
+    return if compte.validation_refusee?
```

Comment:
Je pense que tu peux écrire cela en une ligne : 

```suggestion
    return droits_comptes_refuses if compte.validation_refusee?
```

Reply:
Quand je mets ton changement : 
```
rubocop: warning
Lint/ReturnInVoidContext - Do not return a value in `initialize`.
```